hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/BlockSender.java

```diff
@@ -631,6 +631,7 @@ private int sendPacket(ByteBuffer pkt, int maxChunks, OutputStream out,
          * 
          * Reporting of this case is done in DataXceiver#run
```

Comment:
I am fine with the update though the exception would be threw and processed in `DataXceiver#run`